yellowbrick/features/manifold.py

```diff
@@ -636,7 +642,11 @@ def manifold_embedding(
 
     # Fit and finalize (calls draw)
     viz.fit_transform(X, y)
-    viz.finalize()
+
+    if show:
+        viz.show()
+    else:
+        viz.finalize()
```

Comment:
Hey there @sjainit @naresh-bachwani @bbengfort &mdash; while I was working on building the docs images for this PR, it occurred to me that this pattern of using ```if show: viz.show() else: viz.finalize()``` in the quick methods will mean that for any of the `ProjectionVisualizers`, we don't get a colorbar or colors (since [that gets set in `finalize`](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/yellowbrick/features/projection.py#L271-L297) and all `ProjectionVisualizers` call to `super` inside `finalize`). I've been trying to play around with different combinations of kwargs here but I haven't been able to get them to show any colors or colorbar for the quick method. Am I missing something?